tests/models/templates/test_csms_api.py

```diff
@@ -68,14 +71,18 @@ def manifest_change_setup(cidc_api, monkeypatch):
 
         # insert manifest before we check for changes
         insert_manifest_from_json(deepcopy(manifest), uploader_email="test@email.com")
+        insert_manifest_into_blob(deepcopy(manifest), uploader_email="test@email.com")
         # should check out, but let's make sure
+        validate_json_blob(
+            TrialMetadata.select_for_update_by_trial_id("test_trial").metadata_json
+        )
         validate_relational("test_trial")
 
 
 def test_detect_changes_when_excluded(cidc_api, clean_db, monkeypatch):
     with cidc_api.app_context():
         manifest_change_setup(cidc_api, monkeypatch)
-        manifest = manifests[-1]
+        manifest = manifests[-2]
```

Comment:
@scvannost a comment linking to the test_data would explain this better

Reply:
more clear in next commit